Revit_Core_Engine/Create/View/ViewReflectedCeilingPlan.cs

```diff
@@ -97,6 +97,77 @@ public static View ViewReflectedCeilingPlan(this Document document, Autodesk.Rev
         }
 
         /***************************************************/
+
+        [Description("Creates and returns a new Reflected Ceiling Plan view in the current Revit file.")]
+        [Input("document", "Revit current document to be processed.")]
+        [Input("level", "The level that the created Floor Plan refers to.")]
+        [Input("viewName", "Optional, name of the new view.")]
+        [Input("scopeBoxId", "Optional, the Scope Box to attempt to apply to the newly created view.")]
+        [Input("viewTemplateId", "Optional, the View Template Id to be applied in the view.")]
+        [Input("cropRegionVisible", "True if enable crop region visibility.")]
+        [Input("annotationCrop", "True if enable annotation crop visibility.")]
+        [Output("viewReflectedCeilingPlan", "The new view.")]
+        public static View ViewReflectedCeilingPlan(this Document document, string viewName, Level level, ElementId scopeBoxId = null, ElementId viewTemplateId = null, bool cropRegionVisible = false, bool annotationCrop = false)
```

Comment:
Can't we just use `Revit_Core_Engine\Create\View\ViewPlan.cs` and no longer need this method?

Reply:
Good idea, I will add ViewType input and make it as a one method 👍